### ISSUE TITLE
Incorrect error assigning-non-slot when assigning to a property

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#359]
Originally reported by: **Anonymous**

---

Pylint is misdetecting property assignments as non-slot assignments.

```
#!python

class Foo():
  __slots__ = ("_bar", "_baz")
  bar = property(lambda self: self._bar, lambda self, value: setattr(self, "_bar", value)
  def __init__(self):
      self.bar = 5 # error here
```

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/359
